Code/Tools/SceneAPI/SceneCore/Utilities/DebugOutput.cpp

```diff
@@ -22,6 +23,16 @@
 
 namespace AZ::SceneAPI::Utilities
 {
+    bool IsDebugEnabled()
+    {
+        bool resultValue = false;
+        if (auto* registry = AZ::SettingsRegistry::Get())
+        {
+            registry->Get(resultValue, AZ::SceneAPI::Utilities::Key_AssetProcessorInDebugOutput);
+        }
+        return resultValue;
+    }
+
```

Comment:
Authors note:  The other parts of the system already read/wrote to this existing registry key, for the same actual purpose, but I figure exposing it via a DLL EXPORT would allow simpler code and better linkage since then nobody else has to include the registry header files or care where this comes from.